.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: node_js
+dist: trusty
```

Comment:
Did you change this because of the `xvfb` issue? If so, you can fix tit like this: https://github.com/canjs/can-fixture-socket/pull/76/commits/4eca4ee5446e0bab4893c71895fb5270fa51dfe4.